include/vkgcDefs.h

```diff
@@ -151,12 +151,13 @@ enum class BasicType : unsigned {
 
 /// Enumerates LLPC shader stages.
 enum ShaderStage : unsigned {
-  ShaderStageVertex = 0,  ///< Vertex shader
-  ShaderStageTessControl, ///< Tessellation control shader
-  ShaderStageTessEval,    ///< Tessellation evaluation shader
-  ShaderStageGeometry,    ///< Geometry shader
-  ShaderStageFragment,    ///< Fragment shader
-  ShaderStageCompute,     ///< Compute shader
+  ShaderStageVertex = 0,                                ///< Vertex shader
+  ShaderStageTessControl,                               ///< Tessellation control shader
+  ShaderStageTessEval,                                  ///< Tessellation evaluation shader
+  ShaderStageGeometry,                                  ///< Geometry shader
+  ShaderStageFragment,                                  ///< Fragment shader
+  ShaderStageCompute,                                   ///< Compute shader
+  ShaderStageFetch,                                     ///< Fetch shader for vertex inputs
   ShaderStageCount,                                     ///< Count of shader stages
```

Comment:
I don't know what to do about this.  I changed 1 line and clang-format changed the rest.  If you want to avoid changes like this, we have to change the clang format file.  I don't think it is a good idea for us to pick and choose which clang format change to keep or revert.

Reply:
Okay, that's unfortunate. Just keep it as-is for the purpose of this change, no use fighting clang-format over this.

In the long run, I'd hope we could keep alignment to a minimum, especially in API-related enums where there is a good chance that different feature branches add to the same enum in different ways, and then whitespace changes just cause completely unnecessary merge pain.